src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -298,6 +290,22 @@ $pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
     display: none;
     visibility: hidden;
   }
+
+  .pf-m-chip-container {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: var(--pf-c-data-toolbar__expandable-content--m-chip-container--MarginTop);
+    margin-left: 0;
+
+    .pf-c-data-toolbar__item,
+    .pf-c-data-toolbar__group {
+      margin-right: var(--pf-c-data-toolbar--spacer);
+    }
+
+    .pf-c-data-toolbar__item {
+      margin-top: var(--pf-c-data-toolbar__expandable-content--m-chip-container__item--MarginTop);
```

Comment:
and this to `--pf-c-data-toolbar__expandable-content__group--m-chip-container__item--MarginTop`